Cargo.toml

```diff
@@ -49,6 +49,8 @@ iroha_executor_data_model = { version = "=2.0.0-rc.1.0", path = "crates/iroha_ex
 iroha_test_network = { version = "=2.0.0-rc.1.0", path = "crates/iroha_test_network" }
 iroha_test_samples = { version = "=2.0.0-rc.1.0", path = "crates/iroha_test_samples" }
 
+iroha_swarm = { version = "=2.0.0-rc.1.0", path = "crates/iroha_swarm" }
```

Comment:
How about directly specifying this as a `iroha_kagami` dependency, not as a workspace dependency?

Reply:
You are right. It is not needed here unless used externally.

Reply:
`inquire` also would be removed from workspace dependencies